### PR TITLE
AWS Integration Provider

### LINKED ISSUE #71: Integration of AWS
- [ ] Investigate the integration of AWS 
- [ ] How do we handle Region/Zones (DC only support Datacenter)
- [ ] Check if we create an own AMI with all deps

### PR DESCRIPTION
**Pull request by lbb**
Resolves #71

### CODE REVIEW

File: provider/cloud/aws/aws.go [hunk 1]
```diff
@@ -51,6 +51,15 @@ type aws struct {
 	datacenters map[string]provider.DatacenterMeta
 }
 
+// Init template
+func Init() {
+	tplPath := "template/coreos/cloud-config-node.yaml"
+	if parsed, err := template.New("cloud-config-node.yaml").Funcs(ktemplate.FuncMap).ParseFiles(tplPath); err != nil {
+		tpl = parsed
+		glog.Errorln("template not found:", err)
```
Comment (scruplelesswizard) [inline provider/cloud/aws/aws.go]: The path where the template is being looked for would be helpful

**Comment by scruplelesswizard** [comment]
Where are we at with this?

File: provider/cloud/aws/manual/manual.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Manual to create access to AWS.
+
+## 1. Get your user id.
+* Under your username go to `My Account`.
+    * If you are a fedreated user go to `Security Credentials`.
+    * Go to users and find yourself.
+    * The ARN contains your id `arn:aws:iam::YOUR_USER_ID:user/YOUR_NAME`
+* Remember your ID
+
+## 2. Create a plocy.
```
Comment (scruplelesswizard) [inline provider/cloud/aws/manual/manual.md]: typo

File: handler/datacenter.go [hunk 3]
```diff
@@ -54,6 +59,60 @@ func datacentersEndpoint(
...
+
+		// Create aws ec2 client.
+		config := aws.NewConfig()
+		config = config.WithRegion(dc.Spec.AWS.Region)
+		// TODO(realfake): Implement token for AWS.
+		config = config.WithCredentials(credentials.NewStaticCredentials(req.Username, req.Password, ""))
+		sess := ec2.New(session.New(config))
+		// Begin describing key pairs.
+		keys, err := sess.DescribeKeyPairs(&ec2.DescribeKeyPairsInput{})
+		println("1")
```
Comment (sttts) [inline handler/datacenter.go]: println?

**Comment by sttts** [comment]
lgtm

File: handler/datacenter.go (hunk 3 above)
Comment (scruplelesswizard) [reply handler/datacenter.go]: Debugging. That commit needs to be reverted

**Review by scruplelesswizard — APPROVED** [review]

**Pull request merged**